SecondDeliverable/controller/csvcontroller.go

```diff
@@ -0,0 +1,98 @@
+package controller
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"main/usecase"
+
+	"github.com/gorilla/mux"
+)
+
+// Index ..
+func Index(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintf(w, "Welcome to my Diablo 3 Items-API")
+}
+
+<<<<<<< Updated upstream
```

Comment:
Seems like this was a conflict, you need to decide which change is the valid one, I guess is the second one